README.md

```diff
@@ -33,5 +33,6 @@
 | Name                                    | Awards |
 |-----------------------------------------|-------------|
 | [Bountiful Open Source Summer- BOSS](http://lab.codingblocks.com/boss/) | Prizes for winners*|
+| [India Hacks - Open Source Track](https://www.hackerearth.com/sprints/open-source-india-hacks-2016/) | Prizes for winners|
```

Comment:
👋 they did not have any in 2017?

Reply:
I don't think they did but I have tweeted to them asking whether they will bring it back this year. If not I will close this.